bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/AutomaticInboxProcessor.java

```diff
@@ -78,6 +79,8 @@ public void receiveTypedEvent(ThingStatusInfoChangedEvent event) {
         }
     }
 
+    // InboxListemer methods
```

Comment:
Listener (m => n)

Reply:
Do we need such marker? I don't think we used it on all the other places.

Reply:
No, will remove it. :)